chapter04/ex_4_4.pl

```diff
@@ -0,0 +1,22 @@
+#!perl
+use 5.010;
+
+&greet( "Fred" );
+&greet( "Barney" );
+&greet( "Wilma" );
+&greet( "Betty" );
+
+sub greet {
+    state $last;
+
+    my $name = shift @_;
+
+    if( defined $last ) {
+        printf "Hi %-10s! %s is also here!\n", $name, $last;
```

Comment:
I can see %s, but I'm don't know what %-10s means, it will be nice to explain in details during the meeting.